src/screens/accountSettings/EditUnits.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { View, StyleSheet, ScrollView, FlatList } from "react-native";
+import type { NavigationScreenProp } from "react-navigation";
+
+import { updateAccount } from "../../actions/accounts";
+import UnitRow from "../../components/UnitRow";
+import type { Account } from "../../types/common";
+
+const mapStateToProps = () => ({});
+
+const mapDispatchToProps = {
+  updateAccount
+};
+
+class EditUnits extends Component<{
+  updateAccount: ($Shape<Account>) => *,
+  navigation: NavigationScreenProp<{
+    params: {
+      account: Account
+    },
+    goBack: () => void
```

Comment:
I don't think you need to define that. goBack() is not part of `this.props.navigation.state`, it is part of `this.props.navigation` which directly is typed by `NavigationScreenProp`. 
see https://github.com/react-navigation/react-navigation/blob/e6559f5878995e7fc9dd6ffe9036bbe3f584e54e/flow/react-navigation.js#L488